language-server/modules/langserver-core/src/test/resources/inlayhint/source/inlayhint4.bal

```diff
@@ -0,0 +1,20 @@
+type Company record {
+    string name;
+    string city;
+};
+
+type Employee record {
+    string name;
+    string age;
+};
+
+public isolated client class EmpDetails {
+    isolated remote function addDetails(Company company, Employee employee) {
+
+    }
+}
+
+public function test() {
+    EmpDetails details = new();
+    details->addDetails({city: "Colombo", name: "WSO2"}, {name: "John", age: 24});
+}
```

Comment:
Shall we add tests cases with rest-args?

Reply:
I have already added a test case to cover rest args
https://github.com/ballerina-platform/ballerina-lang/pull/40868/files#diff-06fdf971a01b0c6b2822516c6e00ad9cbae0a8bd4b534e23deedb8fa42444889